.github/workflows/task_runner_docker_e2e.yml

```diff
@@ -66,6 +66,14 @@ jobs:
           pip install .
           pip install -r test-requirements.txt
 
+      - name: Create OpenFL image
+        id: create_openfl_image
+        run: |
+          echo "Creating openfl image. This may take a few minutes..."
```

Comment:
Creating openfl image with current branch and repository.

I think this comment will be more helpful.

Reply:
Done.